File: app/src/main/java/pl/szczodrzynski/edziennik/data/db/full/LuckyNumberFull.java
/*
 * Copyright (c) Kacper Ziubryniewicz 2020-1-6
 */

package pl.szczodrzynski.edziennik.data.db.full;

import pl.szczodrzynski.edziennik.data.db.entity.LuckyNumber;

public class LuckyNumberFull extends LuckyNumber {
    // metadata
    public boolean seen;
    public boolean notified;
    public long addedDate;
}

File: app/src/main/java/pl/szczodrzynski/edziennik/data/db/full/LuckyNumberFull.kt
/*
 * Copyright (c) Kuba Szczodrzyński 2020-4-25.
 */
package pl.szczodrzynski.edziennik.data.db.full

import pl.szczodrzynski.edziennik.data.db.entity.LuckyNumber
import pl.szczodrzynski.edziennik.utils.models.Date

class LuckyNumberFull(
        profileId: Int, date: Date,
        number: Int
) : LuckyNumber(
        profileId, date,
        number
) {
    // metadata
    var seen = false
    var notified = false
}
